### ISSUE TITLE
Introducing Page-Related Attributes for Enhanced Event Association

### ISSUE THREAD

**Issue by Abinet18** [open-telemetry/semantic-conventions#487]
### Introducing Page-Related Attributes for Enhanced Event Association

To enhance our event tracking capabilities on specific web pages, it's imperative that we can associate events or spans that occur on a page with that particular page itself. For instance, when a core web vital event arises, we should be able to link it directly to the page identified in the document load trace. Similarly, for xhr and fetch requests originating from the same page, it's beneficial to link them back to the page trace.

To achieve this, we propose the integration of new attributes: page.trace_id and page.span_id. These attributes will be appended to events and spans, respectively, allowing them to reference the document load trace ID and the root span ID of the page where these activities are taking place.

**Comment by trask** [comment]
cc @open-telemetry/sandbox-web-js-maintainers (and maybe @open-telemetry/swift-approvers and @open-telemetry/android-approvers)

**Comment by MSNev** [comment]
I don't think this is the way to do this, the events may already have the trace id / span id associated with them, so why call out this duplication information???
ie. Logs (I believe) already has the ability to be linked to a span

Lets discuss in the Client RUM Sig
